custom/nic_compliance/tests/test_utils.py

```diff
@@ -53,9 +53,9 @@ def test_no_replay_attack_for_mobile_heartbeat_attempt(self):
             obfuscated_password = "sha256$1e2d5bc2hhMjU2JDFlMmQ1Yk1USXpORFUyZjc5MTI3PQ==f79127="
             client = Client(enforce_csrf_checks=False)
             auth_headers = {
-                'HTTP_AUTHORIZATION': 'Basic ' + base64.b64encode('%s:%s' % (
-                    self.username, obfuscated_password
-                )),
+                'HTTP_AUTHORIZATION': 'Basic ' + base64.b64encode(
+                    ('%s:%s' % (self.username, obfuscated_password)).encode('utf-8')
+                ).decode('utf-8'),
```

Comment:
why is there a need to encode-decode it?

Reply:
In python 3 all encode-decode operations must be explicit, unlike python 2 which will implicitly try them for you.  b64encode takes bytes as its argument (in py2 the encode is implicit) and returns bytes (in py2, concatenation calls decode explicitly on bytes)